internal/testutil/k8s.go

```diff
@@ -0,0 +1,64 @@
+package testutil
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+const servicesFile = "services.yml"
+
+func FakeK8sClient() (*fake.Clientset, error) {
+	fakeCs := fake.NewSimpleClientset()
+
+	k8sServicesYaml, err := testDataDir.ReadFile(filepath.Join(testDataRootDir, servicesFile))
```

Comment:
Why do we focus so much on services? Should we extract this to something like "setupServices".

Reply:
I focused on services because it's the only thing that KSM uses. The idea is to progressively add more things as we need them.

Reply:
Moreover, in upcoming PRs the idea is to generate files like the `services.yaml` automatically.

Reply:
Added also a helper for Namespaces, which for now is empty but will be populated in the follow-up PR.